test/test_integration_cli.py

```diff
@@ -435,8 +435,10 @@ def test_uses_terraform_to_destroy(self, *args):
                 '-backend-config=bucket=tfstate-bucket',
                 '-backend-config=region=us-north-4',
                 '-backend-config=key=terraform.tfstate',
-                '-backend-config=workspace_key_prefix={}'
-                .format(component_name),
+                (
+                    '-backend-config=workspace_key_prefix='
+                    f'your-team/{component_name}'
```

Comment:
`your-team` will this work?

Reply:
Yeah, it's a reference to the fixture data in the dummy `cdflow.yml` file: https://github.com/mergermarket/cdflow-commands/blob/4ae4498420114856b45ecf8c73162c7c8c754394/test/test_integration_cli.py#L48-L52